src/Microsoft.AspNet.Mvc.DataAnnotations/NumericClientModelValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNet.Mvc.DataAnnotations;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    public class NumericClientModelValidator : IClientModelValidator
```

Comment:
Add doc comments for class and the `public` method. (`<inheritdoc />` is fine for the method.)